databases_201.rst

```diff
@@ -25,6 +25,84 @@ CouchDB
 Hadoop
 ------
 
+ElasticSearch
+-------------
+
+**Intro**
+
+*ElasticSearch* main logo - "you know, for search". It's near real-time search engine and document-oriented datastore. It's written in Java, so lots of processes depends on JVM (Java Virtual Machine). System has awesome REST API, its documents and queries - all is represented via JSON (https://en.wikipedia.org/wiki/JSON). All configuration can be set up in elasticsearch.yml file. Analogues: Apache Solr, Sphinx.
```

Comment:
The [Style Guide](http://www.opsschool.org/en/latest/meta/style_guide.html#spacing) recommends splitting long sentences to each on their own line.